pyproject.toml

```diff
@@ -120,3 +120,25 @@ ban-relative-imports = "parents"
 
 [tool.ruff.pydocstyle]
 convention = "pep257"
+
+[tool.setuptools_scm]
```

Comment:
should we use poetry for building as it is configured? 
https://github.com/ansible/eda-server/blob/main/pyproject.toml#L3